jenkins/prebuild.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+rm -rf ~/temp
```

Comment:
Might be worth adding a comment in this file explaining why we're putting it at ~/temp rather than the current directory. Essentially, "pip has a max path length of 126 characters which causes an issue with Jenkins whose paths are very long."